invenio_rdm_records/theme/templates/invenio_rdm_records/details/doi.html

```diff
@@ -28,4 +29,10 @@ <h4>
       {{ badges_formats_list(url_for('invenio_formatter_badges.badge', title='DOI', value=id_doi, ext='svg', _external=True, _scheme='https'), id_doi|pid_url(scheme='doi')) }}
     </div>
   </div>
+{% else %}
+  <span class="get-badge" data-toggle="tooltip" data-placement="bottom" style="cursor: pointer;"
+    title="Get the DOI badge!">
+    No minted DOI
```

Comment:
Do we need it to be in the `get=bage` span? If so we could place the if statement inside the span, otherwise we could remove the span in this case.